include/zephyr/bluetooth/audio/capabilities.h

```diff
@@ -234,10 +234,12 @@ struct bt_audio_capability {
 	uint16_t context;
 	/** Capability codec reference */
 	struct bt_codec *codec;
+#if defined(CONFIG_BT_AUDIO_UNICAST_SERVER)
 	/** Capability preferences */
 	struct bt_audio_capability_pref pref;
 	/** Capability operations reference */
 	struct bt_audio_capability_ops *ops;
+#endif /* CONFIG_BT_AUDIO_UNICAST_SERVER */
 	sys_snode_t node;
```

Comment:
I wonder if `pref` should be moved to a unicast server-only struct since it has no value for the broadcast sink. 

Somewhat the same with the `ops` as those are only used for the unicast server, but the broadcast sink could be refactored to use some of them (except that the broadcast sink always works on a group, and not individual streams).

Reply:
Each BIS is an individual stream I guess and an BASS-client can dynamically instruct a receiver to sync to one or more BISes - hence the application needs to be able to handle individual BIS LC3 start/stop + cleanup.
Don't think it needs to be part of this PR though, unless it is easier? (Not sure how it works today from an API perspective - maybe this is already supported)

Reply:
These are the `struct bt_audio_capability_ops`, which are used by the unicast server for stream configuration. 

The `struct bt_audio_stream_ops` still has the `started` and `stopped` callbacks for the broadcast sink role.